setup.py

```diff
@@ -89,7 +108,7 @@ def get_cffi_modules():
 
 
 setup(
-    cmdclass={'install': CheckCairoXcb},
+    cmdclass=cmdClass,
```

Comment:
Why changing this?

Reply:
because of above - custom Sphinx doc build flow

Reply:
I really don't get it; you just made it a variable, how is it changing anything?

Reply:
I expand it with Sphinx. Look at `cmdClass['build_sphinx'] = BuildDocWrapper`

Reply:
Alright, I get it now!

Do we really want to build the docs when installing Qtile with pip? I think we should rather exclude them, and only have them installed by proper package managers, and the package maintainers will decide what to do with the docs.

I personally see these docs as something that is meant to be hosted online, not as something that should take bytes on every user's computer disk. What do you think?

Reply:
Also, that would make the pip install longer and more prone to errors in case the doc does not build, and I think we don't want this.

Reply:
Tycho asked to include it in docs build. But how documentation building is related to `pip install qtile`? This cmdclass is just commands to build. If you run simple install - it will not generate docs, just with `python setup.py build_sphinx` (if I not mistype something) and I'm fixing that part

Reply:
Alright, I get it now! (second edition)

I didn't know about that `build_sphinx` thing, it's more clear now that I read about it on the Sphinx documentation.